### ISSUE TITLE
Scroll-related Issues with Pod Logs Viewer

### ISSUE THREAD

**Issue by Asuforce** [argoproj/argo-cd#14387]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

After upgrading to Argo CD v2.7.7, there have been changes in the behavior of the pod logs viewer, resulting in the following issues:

- Horizontal scrolling is independent for each line
- Logs that are too wide require downloading to be fully readable

I'm wondering if these changes are intentional specifications or if these are bugs. 
I find it less convenient compared to the previous version.

**To Reproduce**

Check pod logs from the pod log viewer.

**Expected behavior**

- The ability to scroll horizontally across the entire log instead of on a per-line basis.
- Being able to view long logs directly within the pod logs viewer.

**Screenshots**

**Version**

```shell
argocd: v2.7.7+4650bb2.dirty
  BuildDate: 2023-07-05T22:18:46Z
  GitCommit: 4650bb2817c3c81405f40cf77e93ef2b5fb275fb
  GitTreeState: dirty
  GoVersion: go1.20.5
  Compiler: gc
  Platform: darwin/arm64
argocd-server: v2.7.7+4650bb2.dirty
```

**Logs**

```
```

**Comment by crenshaw-dev** [comment]
Maybe fixed by https://github.com/argoproj/argo-cd/pull/14199/files?

**Comment by Asuforce** [comment]
@crenshaw-dev 
Thank you for your sharing. So it is a spec now.

> Logs that are too wide require downloading to be fully readable

How about this problem ?

**Comment by mhoyer** [comment]
I can confirm that this UX is not very convenient anymore. I even can't use my horizontal mouse wheel to scroll at all, though `Shift`+`Wheel` works. So or so, scrolling single log lines feels just strange at all.

**Comment by alexec** [comment]
This may be fixed by #14199. It should fix the need to scroll horizontally long lines; that was my main goal. 

I’m not sure about the `Shift + Wheel` problem. Maybe fixed too.

@crenshaw-dev will this go in v2.7.8?

**Comment by Asuforce** [comment]
@alexec Thank you for your supporting. It fixed on v2.7.9.

**Comment by maartengo** [comment]
This issue seems to have resurfaced in the latest version:

https://github.com/argoproj/argo-cd/assets/46345801/98183b9b-7976-4019-85b4-5a640e1ac49e


```
{
    "Version": "v2.9.1+58b04e5",
    "BuildDate": "2023-11-14T15:08:20Z",
    "GitCommit": "58b04e5e11d007b0518853029ff7612c24a2eb35",
    "GitTreeState": "clean",
    "GoVersion": "go1.21.3",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KustomizeVersion": "v5.2.1 2023-10-19T20:13:51Z",
    "HelmVersion": "v3.12.1+gf32a527",
    "KubectlVersion": "v0.24.2",
    "JsonnetVersion": "v0.20.0"
}
```
